website/blog/2020-06-30-data-lake-analytics-blog.md

```diff
@@ -0,0 +1,127 @@
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/DLA_architecture.png)
+<br><br>
+FrontNode is the SQL access layer which provides MySQL protocol to let users interact with. Since MySQL is a very flexible and compatible service, it becomes a user-friendly connection protocol to be used by DLA users. Unified Meta Layer is a metadata repository which holds ALL metadata for all data sources DLA can support. DLA was forked from PrestoDB Version 0.227 and is now integrated with mainstream data sources in Alibaba Cloud.
+<br><br>
+
+### DLA Product Key Features:
+
+1. Serverless:<br>
+a. No infrastructure and management cost<br>
+b. Zero start-time<br>
+c. Transparent upgrade<br>
+d. QoS Elastic Service<br>
+2. Standard SQL and Connections<br>
+a. Compatible with standard SQL<br>
+b. Rich built-in function support<br>
+c. JDBC/ODBC support<br>
+d. Compatible with BI tools<br>
+3. Heterogeneous Data Source<br>
+a. Enable OSS Data analysis requirements<br>
+b. Enable Table Store Data for SQL querying service<br>
+c. Federated data analysis across multiple database instances<br>
+d. Ease of interconnection analysis for multiple data sources<br>
+e. Any forms of analytics in “Data Lake”<br>
+4. Optimized Computing Engine<br>
+a. Support for unstructured data<br>
+b. Vectorized execution and optimization<br>
+c. Operator pipeline optimization<br>
+d. Resource isolation and priority<br>
+
+
+
+## DLA Use Cases
+
+
+### Typical Data Flow Scenarios:
+
+Generally DLA empowers cloud users analytic capability in 3 typical user data flow scenarios:<br>
+1. The cloud user uploads the business generated data such as Log, CSV, JSON and other files, directly to OSS(Alicloud Object Storage Service), and then uses DLA to directly point to the file or folder for which the table is created to query. Then the user may use BI tools to analyze and visualize business data analytics.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case1.png)
+<br><br>
+2. The cloud user can directly copy and upload data in the Hadoop’s legacy format, such as Parquet, ORC, RCFile, Avro, and other formats, to OSS, then use DLA to directly point files or folders for which the table is created to query. Then the user may use BI tools for business data analysis and visualization. <br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case2.png)
+<br><br>
+3. In order to provide better query performance and low storage cost for subsequent data analysis on OSS, data can be converted into Parquet or ORC format to improve the cost performance of repeated data analysis.<br>
+<br><br>
+![](/img/blog/2020-06-30-data-lake-analytics-blog/case3.png)
+<br><br>
+
+### Alibaba Use Case Studies: 
+
+#### Customer Case #1: Log Data Analytics
+
+Cloud customers generate log data which is stored on OSS. The challenge is that those log data are unstructured and typically they are in Petabytes of data size. <br>
```

Comment:
Maybe
"The challenge is that the log data is unstructured and typically on the order of petabytes in size."

Reply:
fixed, thanks